### ISSUE TITLE
elm-export-persistent-0.1.2 build failure with persistent-2.8

### ISSUE THREAD

**Issue by DanBurton** [jb55/elm-export-persistent#2]
As seen on the Stackage build server:

```
[2 of 3] Compiling Elm.Export.Persist.Ent ( src/Elm/Export/Persist/Ent.hs, dist/build/Elm/Export/Persist/Ent.o )

src/Elm/Export/Persist/Ent.hs:79:50: error:
    • Could not deduce (ToJSON (Key a)) arising from a use of ‘toJSON’
      from the context: (KnownSymbol field, ToJSON a)
        bound by the instance declaration
        at src/Elm/Export/Persist/Ent.hs:76:10-62
    • In the second argument of ‘Map.insert’, namely ‘(toJSON k)’
      In the first argument of ‘Object’, namely
        ‘(Map.insert keyname (toJSON k) hmap)’
      In the expression: Object (Map.insert keyname (toJSON k) hmap)
   |
79 |       Object hmap -> Object (Map.insert keyname (toJSON k) hmap)
   |                                                  ^^^^^^^^
```

I was able to reproduce locally like so:

```bash
stack unpack elm-export-persistent-0.1.2 && cd elm-export-persistent-0.1.2
edit stack.yaml # add the following stack.yaml
stack build
```

```yaml
# stack.yaml
resolver: nightly-2018-03-10
extra-deps:
- persistent-2.8.1
```

**Comment by DanBurton** [comment]
I've removed elm-export-persistent from the Stackage nightly build plan, and unfortunately it won't appear in the initial LTS 11 release.

To get back into nightly, open a PR on the Stackage repo as usual. To get into LTS 11, open an issue on https://github.com/fpco/lts-haskell/issues.

**Comment by jb55** [comment]
Thanks! Fixed in 2294c79bef1fa978d394249d98f584aa48499da0 (published at 0.2.0)
